src/Microsoft.Fast.Components.FluentUI/Components/FluentTextField.razor

```diff
@@ -1,5 +1,20 @@
 @inherits FluentInputBase<string>
-<fluent-text-field autofocus="@Autofocus" appearance="@Appearance.ToAttributeValue()" value="@BindConverter.FormatValue(CurrentValue)" @onchange="@(EventCallback.Factory.CreateBinder<string>(this, __value => CurrentValueAsString = __value, CurrentValueAsString))" readonly="@Readonly" disabled="@Disabled" required="@Required" placeholder="@Placeholder" @attributes="AdditionalAttributes">@ChildContent</fluent-text-field>
+<fluent-text-field autofocus="@Autofocus"
+                   appearance="@Appearance.ToAttributeValue()"
+                   value="@BindConverter.FormatValue(CurrentValue)"
+                   @onchange="@(EventCallback.Factory.CreateBinder<string>(this, __value => CurrentValueAsString = __value, CurrentValueAsString))"
+                   type="@TextFieldType.ToAttributeValue()"
+                   size="@Size"
+                   minlength="@MinLength"
```

Comment:
As a matter of consistency, is it strange that `@MinLength` is PascalCased whereas most of the others are just lowercase?  Would it be an improvement to bring consistency to all?

Reply:
Think you got a point there. Did not touch on the ones that were already there, but would be more consistent to rename `@Readonly` to `@ReadOnly` throughout the library and check potential others